src/views/home/components/transaction-form/transaction-form.view.tsx

```diff
@@ -13,13 +13,12 @@ import tokens from "src/assets/tokens/tokens.json";
 import Button from "src/views/shared/button/button.view";
 import AmountInput from "src/views/home/components/amount-input/amount-input.view";
 import { chains } from "src/constants";
-import { useUIContext } from "src/contexts/ui.context";
 import { useTransactionContext } from "src/contexts/transaction.context";
 import { Chain, Token } from "src/domain";
 
 const TransactionForm: FC = () => {
   const classes = useTransactionFormtStyles();
-  const { openModal, closeModal } = useUIContext();
+  const [openList, onOpenList] = useState("");
```

Comment:
What are this variables and why are you always calling them like: `onOpenList("");`?